integrasjon/README.md

```diff
@@ -28,3 +28,9 @@ Posten tilbyr en tilleggstjeneste for langtidslagring. Denne er nærmere beskrev
 Om langtidslagring er aktivert vil nedlasting av XAdES og PAdES (som beskrevet for hhv. [synkrone](synkron.md#steg-4-laste-ned-pades-eller-xades) og [asynkrone](asynkron.md#steg-3-laste-ned-pades-eller-xades) oppdrag) være tilgjengelig i 50 år.
 
 Benyttes ikke tilleggstjenesten for langtidslagring, slettes XAdES og PAdES 30 dager etter signering.
+
+### Sletting fra arkivet
+
+Dokumenter kan slettes fra arkivet på forespørsel. Begrensninger er dokumentert i den [funksjonelle dokumentasjonen](http://digipost.github.io/signature-api-specification/v1.0/#sletting-av-dokumenter).
+
+Når du henter en statusoppdatering (`direct-signature-job-status-response` for synkrone oppdrag, og `portal-signature-job-status-change-response` for asynkrone oppdrag) vil du få en lenke (`delete-documents-url`) du kan gjøre et `DELETE`-request til for å slette dokumentene dersom alle undertegnere har signert dokumentet. 
```

Comment:
"`..dersom alle undertegnere har signert dokumentet`" Betyr det at hvis ikke alle har signert så kan jeg ikke slette det så lenge oppdraget er aktivt, eller gjelder det fram til det utløper eller avbrytes?

Reply:
Hvis ikke alle har signert kan du ikke slette dokumentene så lenge det er aktivt, nei. Men hvis oppdraget utløper eller avbrytes får du heller ikke lov til å slette dokumentene. De vil automatisk bli slettet etter en tid.

Featuren er først og fremst laget for virksomheter som har skrudd på langtidsarkivering av signerte dokumenter :)

Reply:
Right! Seff, det gir mening. Takk

Reply:
Jeg synes at det du skriver bør tas med i dokumentasjonen/gjøres litt mer tydelig. Det står jo `kan slettes fra arkivet på forespørsel`, men jeg forstod det ikke helt umiddelbart.

Reply:
Enig i at det bør inn. Kan slenge inn en setning i [den funksjonelle dokumentasjonen](https://digipost.github.io/signature-api-specification/v1.0/#langtidslagring) hvor jeg føler det hører bedre hjemme.

Reply:
https://github.com/digipost/signature-api-specification/pull/81